anitya/templates/project_new.html

```diff
@@ -119,10 +121,19 @@
     if (str == 'custom'){
       $('#regex_row').show();
       $('#version_url_row').show();
+      $('#regex').attr('placeholder', 'Leave empty for default regex');
     };
 
     $('#example_txt').html(examples[str]);
 
+    if (str == 'custom'){
+      $('#example_txt').append(
+        '<br /> More information in the '
+        + '<a href="{{ url_for("about") }}#test-you-regex">'
+        + 'about#test-you-regex</a>'
```

Comment:
I did, but url_for will not show the full url, it will just give /about, which I thought was even worse than just about#...